### PR TITLE
TS-4641 Changes default for proxy.config.cache.hostdb.sync_frequency …

### PR DESCRIPTION
**Pull request by zwoop**
…to 0

### CODE REVIEW

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/772/ for details.

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/Github-Linux/668/ for details.

**Comment by bryancall** [comment]
👍 - Looks good

File: doc/admin-guide/files/records.config.en.rst [hunk 1]
```diff
@@ -2510,12 +2510,15 @@ HostDB
 
    The filename to persist hostdb to on disk.
 
-.. ts:cv:: CONFIG proxy.config.cache.hostdb.sync_frequency INT 120
+.. ts:cv:: CONFIG proxy.config.cache.hostdb.sync_frequency INT 0
 
-   Set the frequency (in seconds) to sync hostdb to disk.
+   Set the frequency (in seconds) to sync hostdb to disk. The default ``0``
+   turns off HostDB disk writing completely, to enable this set this value to
```
Comment (jpeach) [inline doc/admin-guide/files/records.config.en.rst]: Make this into 2 sentences. It doesn't quite make sense.

**Comment by jacksontj** [comment]
IIRC the plan was to leave this enabled by default? From what I remember of the conversations at the last summit-- the main problem with the current one is that its terrible and causes problems. This new one doesn't have a noticeable performance cost at all, so I'm not sure why we'd want to disable it by default.

**Comment by atsci** [comment]
FreeBSD build *successful*! See https://ci.trafficserver.apache.org/job/Github-FreeBSD/1273/ for details.

**Comment by atsci** [comment]
Linux build *successful*! See https://ci.trafficserver.apache.org/job/Github-Linux/1168/ for details.

**Pull request closed without merging**